collector/os.go

```diff
@@ -131,6 +140,16 @@ func (c *OSCollector) collect(ch chan<- prometheus.Metric) (*prometheus.Desc, er
 		float64(dst[0].FreePhysicalMemory*1024), // KiB -> bytes
 	)
 
+	time := dst[0].LocalDateTime
+	timeZoneName, _ := time.Zone()
+	
+	ch <- prometheus.MustNewConstMetric(
+		c.LocalDateTime,
+		prometheus.GaugeValue,
+		float64(time.Unix()),
```

Comment:
Can you clarify, is this local time or is it unix time? If local time, what does that mean?